### ISSUE TITLE
[BUG] kirby form field not handling multiline 

### ISSUE THREAD

**Issue by cociopigen** [kirbydesign/designsystem#2393]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
 
input field jumps into label, only when there is more than one line in the label. 

https://user-images.githubusercontent.com/22195165/177318908-8c18b344-73df-42ee-a223-d0e75cb4461d.MP4

## Describe how to reproduce the bug

on a phone or screen under 520 width

1. Go to 'Formål og omfang' 
2. Go to page about 'Formål og konti' 
3. Navigate to question ' Har du konti eller investeringer hos andre banker?'
4. Choose  option' Ja'  
5. Click on the input field , keep an eye on the label for the form field.
6. See Error 

## Which Kirby version was used?
 
6.0.1

## What was the expected behavior?

Input field not jumping on multi line label. ( 2 lines at least)

## Add any screenshots

![Screenshot 2022-07-05 at 09 27 31](https://user-images.githubusercontent.com/22195165/177321048-6365bf4b-50c7-4601-aefc-4ba9a2a6a68d.png)
![Screenshot 2022-07-05 at 09 28 15](https://user-images.githubusercontent.com/22195165/177321149-7999e462-8210-4b6c-b254-d05f22c1ae32.png)

## Please complete the following information:
- OS: MacOS Monterey 12.4
- Browser: Chrome
- Browser version: 103.05060.53

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
